res/css/views/messages/_ViewSourceEvent.scss

```diff
@@ -36,7 +36,6 @@ limitations under the License.
         visibility: hidden;
         // override styles from AccessibleButton
         border-radius: 0;
-        padding: 0;
```

Comment:
`.mx_ViewSourceEvent_toggle` is applied to `AccessibleButton` with `kind='link'` on `ViewSourceEvent.tsx`.